### DISCUSSION TITLE
Do I install directly into my OBS folder or into its own standalone folder?

### DISCUSSION THREAD

**Discussion by MajesticalDiscomfort** [Xaymar/obs-stream-effects#971] [Q&A] [answered]
Sorry, I haven't installed OBS plugins before and none of my friends have either :/

**Comment by Xaymar** [comment, marked as answer]
v0.11 and earlier required installation into the OBS Studio directory. From v0.12 onwards this is no longer required, and the installer automatically picks the correct directory.

**Reply by MajesticalDiscomfort** [reply]
I actually have two different versions of OBS on my PC--the most recent version, and the steam version, which I've intentionally backported to version 27.2 in order to be compatible with streamfx. I installed the plugin to the obs folder inside the steamapps folder, but it doesn't appear to work.

**Reply by ypurpl** [reply]
Go back to OBS 28 and get the latest release of StreamFX.
**Make sure to install with admin rights, or it won't work with the Steam release!** (should work with no rights on a normal release however)

**Reply by Xaymar** [reply]
> I actually have two different versions of OBS on my PC--the most recent version, and the steam version, which I've intentionally backported to version 27.2 in order to be compatible with streamfx. I installed the plugin to the obs folder inside the steamapps folder, but it doesn't appear to work.

You will need the matching OBS Studio version for the StreamFX version you wish to use: v0.11.1 is 27.2.x, v0.12.0 is 28.0/28.1. Only v0.12.0 supports Steam, as v0.11.1 modifies the OBS Studio installation directory and as such will be deleted by Steam when verifying or updating. If you do have to use the install-into-obs method, you should always pick the directory that contains bin, data and obs-plugins.
